libs/datasets/sources/can_scraper_helpers.py

```diff
@@ -21,7 +22,11 @@
 
 # Airflow jobs output a single parquet file with all of the data - this is where
 # it is currently stored.
-PARQUET_PATH = "data/can-scrape/can_scrape_api_covid_us.parquet"
+GCS_PARQUET_PATH = (
+    "https://storage.googleapis.com/can-scrape-outputs/final/can_scrape_api_covid_us.parquet"
+)
+
+LOCAL_PARQUET_PATH = dataset_utils.DATA_DIRECTORY / "can_scrape_api_covid_us.parquet"
```

Comment:
I think we'll need to configure *.parquet files to be stored in LFS in .gitattributes.

Maybe go ahead and check the current parquet file in with this PR to make sure that it ends up in LFS?

Reply:
good call,

added this and a second pipeline run at 10 UTC. The `git push` with LFS is taking a long time :/